Raw reference/identifier conflict

vanessa-rodrigues opened the pull request.

maxbrunsfeld commented:
@ObserverOfTime Do you know what's up with the node bindings test failure? Do we need to regenerate the node bindings?

ObserverOfTime commented:
You need to upgrade the tree-sitter dependency.

maxbrunsfeld commented:
It looks like some examples are failing to parse now.

vanessa-rodrigues commented:
Yes, I am trying to reproduce it, but these tests seem to fail even before my changes.

vanessa-rodrigues commented:
Updates: 
- Removed `visibility_modifier` from `foreign_mod_item` and added optional `unsafe`. [ref](https://doc.rust-lang.org/reference/items/external-blocks.html#railroad-ExternalItem)
```bash
  |
6 | pub extern "C" {
  | ^^^ help: remove the qualifier
  |
  = note: place qualifiers on individual foreign items instead
```
- Excluded two files from ci tests, [file1](https://github.com/rust-lang/libc/blob/main/ctest/templates/test.rs), [file2](https://github.com/rust-lang/libc/blob/main/ctest/tests/input/invalid_syntax.rs).

vanessa-rodrigues commented:
@maxbrunsfeld, the tests passed. Can we merge this?

maxbrunsfeld commented:
Thanks!
